crates/ruff_python_formatter/src/expression/expr_lambda.rs

```diff
@@ -9,13 +12,30 @@ use rustpython_parser::ast::ExprLambda;
 pub struct FormatExprLambda;
 
 impl FormatNodeRule<ExprLambda> for FormatExprLambda {
-    fn fmt_fields(&self, _item: &ExprLambda, f: &mut PyFormatter) -> FormatResult<()> {
-        write!(
-            f,
-            [not_yet_implemented_custom_text(
-                "lambda NOT_YET_IMPLEMENTED_lambda: True"
-            )]
-        )
+    fn fmt_fields(&self, item: &ExprLambda, f: &mut PyFormatter) -> FormatResult<()> {
+        let ExprLambda {
+            range: _,
+            args,
+            body,
+        } = item;
+
+        write!(f, [text("lambda")])?;
+
+        if args.args.is_empty() {
+            write!(f, [text(":")])?;
+        } else {
+            write!(
+                f,
+                [
+                    space(),
+                    args.format()
+                        .with_options(ArgumentsParentheses::SkipInsideLambda),
+                    text(":"),
+                ]
+            )?;
+        }
+
+        write!(f, [space(), body.format()])
```

Comment:
Nit (I'm fine with either)

```suggestion
        write!(f, [text("lambda")])?;

        if !args.args.is_empty() {
            write!(
                f,
                [
                    space(),
                    args.format()
                        .with_options(ArgumentsParentheses::SkipInsideLambda),
                ]
            )?;
        }

        write!(f, [text(":"), space(), body.format()])
```

Reply:
Yours is better for sure. 7473457ecaae65ac9b645262bb7587269f71a232